fineract-provider/src/test/java/org/apache/fineract/common/Utils.java

```diff
@@ -28,13 +28,11 @@
 import com.jayway.restassured.path.json.JsonPath;
 import com.jayway.restassured.specification.RequestSpecification;
 import com.jayway.restassured.specification.ResponseSpecification;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.Locale;
-import java.util.Random;
 import org.apache.commons.lang.StringUtils;
 import org.apache.http.conn.HttpHostConnectException;
 
+
+
 /**
  * Util for RestAssured tests. This class here in src/test is copy/pasted :(
  * from and identical to the one in src/integrationTest; please keep it in sync.
```

Comment:
@percyashu do you want to do the same clean up in the same class in src/integrationTest as well?

Reply:
@vorburger src/integrationTest seems to be already clean since all methods are used in integration unlike those copied into src/test